ci/ansible/roles/pulp-test-populate/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Start Pulp
```

Comment:
Ok, I'll try this out.

Can you share any insight you have on how variables are handled when using this method?

i.e. if options are passed `-e myvar='foo'` this is available at the global scope, right? 

I.e. if tasks are run because one role declares dependency on another, then the variable defined on the command line is available to all tasks, right?